library/ajax/index_data.php

```diff
@@ -14,7 +14,7 @@
 global $bb_cfg, $lang, $userdata, $datastore;
 
 $mode = (string)$this->request['mode'];
-$html = '';
+$html = array ();
```

Comment:
```suggestion
$html = [];
```
Лучше в таком формате.